cishouseholds/pipeline/survey_responses_version_2_ETL.py

```diff
@@ -51,9 +58,104 @@ def transform_survey_responses_generic(df: DataFrame) -> DataFrame:
     df = convert_barcode_null_if_zero(df, "blood_sample_barcode")
     df = assign_taken_column(df, "swab_taken", reference_column="swab_sample_barcode")
     df = assign_taken_column(df, "blood_taken", reference_column="blood_sample_barcode")
+    # df = assign_true_if_any(
+    #    df=df,
+    #    column_name_to_assign="sympt_covid_cghfevamn",
```

Comment:
These are the stata variable names, please update them to use the to-be names from the master file. (Sorry, the specification does include the wrong ones used here)

Reply:
renamed